data-products-dp0-2/data-processing.rst

```diff
@@ -94,14 +94,13 @@ and (5) performing forced measurements in each band using the reference band pos
 This last step produces a catalog of independent per-band object measurements that are provided to science users.
 
 
-.. _Data-Processing-Overview-Postprocessing:
+.. _DATA-Processing-Overview-Difference-Imaging:
 
-Post-processing
-===============
+Difference Image Analysis (DIA) production
+==========================================
 
-The DESC DC2 data were simulated and processed using v19.0.0 of the LSST Science Pipelines which relied on the "Gen2" Butler.
-For release as part of DP0.2, these data products were converted to the "Gen3" Butler system and ingested into the Interim Data Facility (IDF).
-Since this was a post-processing conversion rather than a native processing with the Gen3 Butler, there may be some slight inconsistencies in the data structure that raise warnings when accessed by the user (see the :doc:`/data-access-analysis-tools/rsp-warnings`).
-This should be corrected in DP0.2 when the DC2 data will be natively reprocessed with the Gen3 Butler.
-We also note that the post-processing organization of the DC2 catalogs described in `Section 4 of DC2 Data Release Note <https://arxiv.org/abs/2101.04855>`_ have not been applied to the DP0.2 data.
-These data are instead intended to be accessed from the Rubin Observatory TAP service.
+The version (23.0.1) of the LSST Science Pipelines that was used for DP0.2 processing now includes portions of the Prompt Processing pipelines, which centers on template image subtraction and transient detection on the resulting "difference images." 
+Each science observation is subtracted from a template coadd image covering the same area of the sky (in LSST survey operations, templates will be created from the previous year of observations).
+The template images are resampled to the coordinate system of the science image, then convolved with a kernel to produce an image whose PSF matches that of the new science image.
+Once the image subtraction has been performed, similar algorithms to those used in Data Release Processing are used to detect and measure objects on the resulting "difference image." These source detections and measurements make up the _DIASource_ catalog.
+Like the Object table from standard processing, the _DIAObject_ table consists of a superset of unique sources that were detected on all of the difference images. 
```

Comment:
DiaObjects are directly derived from DiaSources, while Objects are not derived at all from Sources.  We should avoid suggesting similarities between the two pairs.

Reply:
I made a small update to the last sentence, to clarify.